src/desktop/apps/artwork/helper.js

```diff
@@ -1,15 +1,42 @@
-import { pickBy, identity } from "lodash"
+import { identity, map, pickBy, trim } from "lodash"
+import { toSentence } from "underscore.string"
 const { APP_URL } = require("sharify").data
 
+const AVAILABILITY = {
+  "for sale": "https://schema.org/InStock",
+  sold: "https://schema.org/OutOfStock",
+}
+
 export const compactObject = o => {
   return pickBy(o, identity)
 }
 
+const displayPrice = artwork => {
+  const { is_price_hidden, is_price_range, price, sale_message } = artwork
+
+  if (is_price_range && !is_price_hidden) {
+    const minAndMaxPrice = price.split("-")
```

Comment:
add a check in case price is an `""` for instance

Reply:
Good catch!